micro-core/src/test/java/app/async/com/aol/micro/server/SimpleApp.java

```diff
@@ -20,4 +27,120 @@ public String myEndPoint(){
 		return "hello world!";
 	}
 	
+	public void test2(){
```

Comment:
Better test naming would be appreciated, so does to follow code style and avoid this indentation or big empty gaps/lines.